source/tests/SGM/SGMEnergyTrajectoryTestView.cpp

```diff
@@ -366,7 +367,8 @@ void SGMEnergyTrajectoryTestView::setTrajectoryPlotData(SGMGratingSupport::Grati
 		                               -1.53,
 		                               1000,
 		                               -5598,
-		                               25.07);
+		                               25.07,
+		                               SGMUndulatorSupport::FirstHarmonic);
```

Comment:
I created this view initially to give me a visual feedback as to how the various optimization calculations vary as a function of energy. Really this view isn't super useful any more, but I kept it in, in order to act as simple check. In reality they should be expanded to show plots for both harmonics, but this didn't seem like a huge priority.